recipes/scikit-posthocs/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and pasting from another recipe, but not really needed.
+
+package:
+  name: scikit-posthocs
+  version: "0.4.0"
+
+source:
+  url: https://github.com/huidongchen/scikit-posthocs/archive/master.zip
```

Comment:
Wait is this pulling from your fork? We cannot do that. We need to move any patch to master or store it and apply it here.

Reply:
Yeah, the original setup file caused some errors during the building. So I had to modify that part to make it work. We are currently kind of rushing to finish a bioconda package which is dependent on scikit-posthocs. Should I contact the package 'scikit-posthocs' developer and wait for them to update it? Or are there any other faster solutions you would suggest?

Reply:
We add a file to repo that patches the source before installation. See the conda build documentation. 

This would be a file with the text produced by git diff between your commit and the upstream repo.